var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -27,27 +27,141 @@ class Trilinos(Package):
     depends_on('matio')
     depends_on('glm')
     depends_on('swig')
+    depends_on('metis')
+    depends_on('suite-sparse')
+    depends_on('tbb')
```

Comment:
I don't think that Trilinos supports tbb. The deprecated version of Kokkos did but the new one doesn't.